webpack.config.js

```diff
@@ -1,27 +1,59 @@
 const webpack = require('webpack');
-const {join} = require('path');
+
+class FallbackRequireWebpackPlugin {
+  apply(compiler) {
+	compiler.hooks.compilation.tap('FallbackRequireWebpackPlugin', (compilation) => {
+		const {renderRequire} = webpack.javascript.JavascriptModulesPlugin.getCompilationHooks(compilation);
+		renderRequire.tap(
+			'FallbackRequireWebpackPlugin',
+			(content, renderContext) => content.replace(
+				/\s*return module\.exports;\s*/,
+				'\nreturn (module.exports && module.exports.name === "webpackEmptyContext") ? require(moduleId) : module.exports;',
+			),
+		);
+	});
+  }
+}
 
 module.exports = {
+	mode: 'production',
```

Comment:
Production already does the minifying, so I moved it to the top (convention)

Reply:
Totally game for this change. For my own reference in the future - is this convention from something documented? I've not run into any kind of docs for webpack.config.js convensions and would love to read/learn more if there's anything documented

Reply:
Not the most documented, but the production flag is automatically set, and when not using the development mode, you can leave off the defaults

https://webpack.js.org/configuration/mode/#mode-production